### DISCUSSION TITLE
IOS Player

### DISCUSSION THREAD

**Discussion by Uncurse** [thatskymod/Sky-CotL-Scripts#54] [Q&A]
I don't know which ones are suitable for IOS and need some help. I know I can't use Canvas but what about the others and how do I set them up?

**Comment by Uncurse** [comment]
also if anyone is aware of any telegrams that share scripts please let me know

**Comment by SkyOwl20** [comment]
I would love to know too please!

**Comment by thami-crtl** [comment]
does someone knows abt any sky mod for ios???

**Reply by thami-crtl** [reply]
did u find some mod for ios ??

**Comment by Slingexe** [comment]
As of March 17th 2025, There are currently no mods for Sky on IOS. If you have a jailbroken IOS device you may have a shot at trying to load scripts. (I am not an IOS user so I don know if this is possible)
